### PR TITLE
Feature/message styles

### PR DESCRIPTION
**Pull request by DJMyles**

### CODE REVIEW

File: css/ui-kit.css [hunk 1]
```diff
@@ -5802,3 +5803,73 @@ article h1:first-of-type, article .hero .site-title:first-of-type, .hero article
...
+
+There are 7 classes for drawing the user's attention:
+
+- `.callout`
+- `.callout--warning` for warnings
+- `.callout--error` for errors
+- `.callout--info` for information
+- `.callout--success` for success
+- `.callout-status` for status
+
```
Comment (Feng-Shui) [inline css/ui-kit.css:5818]: Do we need to remove the reference to `.callout-status`?

**Comment by rujiali** [comment]
Looks good to me. Merge to master now.

**Comment by Feng-Shui** [comment]
@rujiali @DJMyles see above comment re `.callout-status` did we need to tidy that up?

File: css/ui-kit.css (hunk 1 above)
Comment (rujiali) [reply css/ui-kit.css:5818]: Yeah, since `.callout-status` is not available any more.

**Pull request merged**